### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by sison54** [eclipse/paho.mqtt.cpp#368]
It seems that the documentation here, is out of date: 
[https://www.eclipse.org/paho/files/cppdoc/index.html](https://www.eclipse.org/paho/files/cppdoc/index.html)

How do we keep this up to date?

**Comment by fpagliughi** [comment]
Yeah, for some reason I lost write access to the Eclipse pages other than GitHub a while back. I'll get in touch with them to see if I can get it resolved. Having the API docs on-line is helpful.
